### PR TITLE
Adding tacokit@3.0.0 targetting Cordova@5.4.1. tacokit@3.0.0 is compr…

### PR DESCRIPTION
**Pull request by nisheetjain**
…ised of pinned cordova android/ios/windows/wp8 platforms and core plugins matching cordova@5.4.1. tacokit@\

3.0.0 will allow users to seamlessly target cordova@5.4.1 without having to worry about "latest" breaking changes in core cordova plugins`

### CODE REVIEW

File: src/taco-kits/TacoKitMetadata.json [hunk 1]
```diff
@@ -281,6 +279,90 @@
...
+        "tacokit@3.0.0": {
+            "cordova-cli": "5.4.1",
+            "releaseNotesUri": "http://go.microsoft.com/fwlink/?LinkId=722537",
+            "default": true,
+            "platforms": {
+                "android": {
+                    "version": "4.1.1"
+                },
+                "ios": {
+                    "version": "3.9.0"
```
Comment (digeff) [inline src/taco-kits/TacoKitMetadata.json]: It's slightly weird that 2.0.0 has ios: 3.9.1 and here we go back to 3.9.0. Why are we using 3.9.0 here?

**Comment by nisheetjain** [comment]
@meenakb @digeff can you guys take a look?

File: src/taco-kits/TacoKitMetadata.json [hunk 2]
```diff
@@ -116,7 +116,7 @@
         },
         "tacokit@1.1.0": {
             "cordova-cli": "5.1.1",
-            "releaseNotesUri": "http://go.microsoft.com/fwlink/?LinkId=623759",
+            "releaseNotesUri": "http://go.microsoft.com/fwlink/?LinkId=722537",
```
Comment (meenakb) [inline src/taco-kits/TacoKitMetadata.json]: Both the links seem to point to the same page?

File: src/taco-kits/TacoKitMetadata.json (hunk 1 above)
Comment (meenakb) [inline src/taco-kits/TacoKitMetadata.json]: Cordova 5.4.1 seems to pin to ios v3.9.2

File: src/taco-kits/TacoKitMetadata.json [hunk 3]
```diff
@@ -281,6 +279,90 @@
...
+                },
+                "ios": {
+                    "version": "3.9.0"
+                },
+                "windows": {
+                    "version": "4.0.0"
+                },
+                "wp8": {
+                    "version": "3.8.1"
+                }
```
Comment (meenakb) [inline src/taco-kits/TacoKitMetadata.json]: Is this a conscious decision? Cordova 5.4.1 seems to pin Windows 4.1.0 and Wp8 3.8.2.

File: src/taco-kits/TacoKitMetadata.json (hunk 1 above)
Comment (nisheetjain) [inline src/taco-kits/TacoKitMetadata.json]: fixed...

File: src/taco-kits/TacoKitMetadata.json (hunk 2 above)
Comment (nisheetjain) [inline src/taco-kits/TacoKitMetadata.json]: the link points to a page which describes point in time kits

File: src/taco-kits/TacoKitMetadata.json (hunk 3 above)
Comment (nisheetjain) [inline src/taco-kits/TacoKitMetadata.json]: fixed

**Comment by meenakb** [comment]
:+1:

**Comment by digeff** [comment]
:+1:

**Pull request merged**